### PR TITLE
create layout for navigation

### PR DESCRIPTION
**Pull request by anhzin**

### CODE REVIEW

File: css.css [hunk 1]
```diff
@@ -15,8 +15,10 @@ body {
 
 .page .header {
```
Comment (techcoaching) [inline css.css:16]: I think, we should also insert css for page (.page{}) event if it was empty

File: css.css [hunk 2]
```diff
@@ -64,4 +66,49 @@ body {
...
+
+/*navigation*/
+
+.navigation {
+    width: 50%;
+    height: 70px;
+    display: table-cell;
+}
+
+.navigation ul {
```
Comment (techcoaching) [inline css.css]: we should be be able to use ul in css file. Please goes from the UI and analyse the name for that element and use that name in css file

File: index.html [hunk 3]
```diff
@@ -10,10 +10,20 @@
     <div class="page">
         <!--header-->
         <div class="header">
+            <div class="left">
```
Comment (techcoaching) [inline index.html]: in this context, "left" was not clear. reader did not understand what is left. it should be the name of component, more meaningful please

File: index.html [hunk 4]
```diff
@@ -10,10 +10,20 @@
...
             <div class="branch">
                 <img src="logo.png" alt="" class="logo" />
                 <span class="name"> Prechu </span>
             </div>
+            </div>
+            <div class="navigation">
+                <ul>
```
Comment (techcoaching) [inline index.html]: what is the role of ul in navigation context? if it does not play any role, it means, we can remove it

File: index.html [hunk 5]
```diff
@@ -10,10 +10,20 @@
...
+                    <li><span class="item active">Home</span></li>
```
Comment (techcoaching) [inline index.html]: We should have the css name for each li

**Review by techcoaching — CHANGES_REQUESTED** [review]

File: css.css (hunk 1 above)
Comment (anhzin) [reply css.css:16]: Dear Mr Tu!
I don't understand in this case. Can you help me for make it detail?

**Pull request closed without merging**